xds/internal/resolver/helpers_test.go

```diff
@@ -78,17 +78,32 @@ var wantDefaultServiceConfig = fmt.Sprintf(`{
    }]
  }`, defaultTestClusterName, defaultTestClusterName)
 
-// buildResolverForTarget builds an xDS resolver for the given target. It
-// returns the following:
+// buildResolverForTarget builds an xDS resolver for the given target. If
+// the bootstrap contents are provided, it build the xDS resolver using them
+// otherwise, it uses the default xDS resolver.
+//
+// It returns the following:
 // - a channel to read updates from the resolver
 // - a channel to read errors from the resolver
 // - the newly created xDS resolver
-func buildResolverForTarget(t *testing.T, target resolver.Target) (chan resolver.State, chan error, resolver.Resolver) {
+func buildResolverForTarget(t *testing.T, target resolver.Target, bootstrapContents []byte) (chan resolver.State, chan error, resolver.Resolver) {
 	t.Helper()
 
-	builder := resolver.Get(xdsresolver.Scheme)
-	if builder == nil {
-		t.Fatalf("Scheme %q is not registered", xdsresolver.Scheme)
+	var builder resolver.Builder
+	var err error
```

Comment:
Nit: Please move this variable definition to the scope where it is really needed.